pkg/policy/api/rule.go

```diff
@@ -252,3 +256,25 @@ type PortRuleHTTP struct {
 	// +optional
 	Headers []string `json:"headers,omitempty"`
 }
+
+// PortRuleKafka is a list of Kafka protocol constraints. All fields are
+// optional, if all fields are empty or missing, the rule does not have any
+// effect.
+//
+type PortRuleKafka struct {
+	// ApiVersion is an POSIX regex matched against the api version of the
+	// Kafka message
```

Comment:
It may be worth specifying that the API version is always "0" or a string representing a positive integer.